README.md

```diff
@@ -49,10 +49,11 @@ There are a few extra features, controlled by flags, enumerated below.
     <summary>CLI options</summary>
 fsbp-fix takes a subcommand and up to 3 flags:
 
-- **profile**: _Required._ The profile to use when connecting to AWS.
-
 - **region**: _Required._ The region you want to search in.
 
+- **profile**: _Optional._ The profile to use when connecting to AWS. If not
+specified, it will run in all enabled regions.
```

Comment:
I'm confused about the relationship between profiles and regions.  How is it the profile that's optional and not the region(s)?